vector/src/main/java/fr/gouv/tchap/features/login/TchapLoginActivity.kt

```diff
@@ -181,7 +181,8 @@ open class TchapLoginActivity : VectorBaseActivity<ActivityLoginBinding>(), Unlo
         if (loginViewState.isUserLogged()) {
             val intent = HomeActivity.newIntent(
                     this,
-                    accountCreation = loginViewState.signMode == SignMode.SignUp
+                    accountCreation = loginViewState.signMode == SignMode.SignUp,
+                    firstStartMainActivity = false
```

Comment:
Not sure about this, I have to double check

Reply:
Seems good: https://github.com/tchapgouv/tchap-android/commit/b294c9a1fd82a1ef23e618079109c979e6d07770#diff-e39e974dc940da7a8c652504ffbfb1506a3296fc6ceb2c0c28f5524a1e900f5fR303